Ogutu-Brian/Questioner
# #163680803 Render Upcoming meetups to Questioner UI

### What does this PR do?
Allows users to view upcoming meetups from the user interface
### What are the tasks to be completed?
* Ensure the API responses are correctly captured and rendered as feedback to user
* Ensure the user experience is not compromised
### How should this be manually tested?
* After cloning the repo
  `git checkout ft-upcoming-meetups-163680803`
* Using a web browser, launch meetups.html file to see upcoming meetups
### Screenshots
![screenshot from 2019-02-04 06-23-52](https://user-images.githubusercontent.com/30015297/52188728-7ad2ca80-2845-11e9-9241-a006e9d20e32.png)

### What are the relevant pivotal tracker stories?
[#163680803](https://www.pivotaltracker.com/story/show/163680803)
